server/migrations/20181106124148-create-article.js

```diff
@@ -22,6 +22,11 @@ export default {
       type: Sequelize.STRING,
       allowNull: false
     },
+    rating: {
+      type: Sequelize.FLOAT,
+      defaultValue: 0,
```

Comment:
set the default value to null.

 - A user can rate an article 0.
 - An unrated article doesn't have a rating of 0.

Two Different things

Reply:
> set the default value to null.
> 
> * A user can rate an article 0.
> * An unrated article doesn't have a rating of 0.
> 
> Two Different things

This is fixed now.